src/scss/features/_nav.scss

```diff
@@ -41,6 +41,7 @@
 
 		&:first-child {
 			padding-left: 0;
+			// @include oTypographyBold(sans); //TODO: enable when the myFtNavigationV2 flag has been switched on permanently -> related to GDPR
```

Comment:
I think this todo should be for a subnav item?

Reply:
Nope, this is for the 'Portfolio' link, which sits in the primary nav. The sign out/ my account will be in the subnav